lib/src/game/camera.dart

```diff
@@ -0,0 +1,289 @@
+import 'dart:math' as math;
+import 'dart:ui' show Rect;
+
+import '../../components.dart';
+import '../../game.dart';
+
+/// Utility method to smoothly transition vectors.
+void _moveToTarget(
+  Vector2 current,
+  Vector2 target,
+  double ds,
```

Comment:
Maybe use a non-abbreviated name and it'll be easier to read

Reply:
like delta space? I think ds is a pretty standard abbreviation, like dt, we use it all over

Reply:
Or make a comment about it?
I didn't know it was standing for delta space at least, even though I understood what it did when reading the code.

Reply:
I will add one. maybe it can be called displacement